hydroflow_lang/src/graph/ops/multiset_delta.rs

```diff
@@ -5,30 +5,43 @@ use super::{
     WriteContextArgs, RANGE_0, RANGE_1,
 };
 
-// TODO(mingwei): more doc
-/// Multiset delta from the previous tick.
+/// The multiset inverse of [`persist()`](#persist).
+///
+/// > 1 input stream of `T`, 1 output stream of `T`, where `T: Eq + Hash`
+///
+/// For set semantics, [`unique()`](#unique) can be thought of as a "delta" operator, the inverse
+/// of [`persist()`](#persist). In `persist`, new items come in, and all items are repeatedly
+/// released out. Conversely, `unique` take repeated items in, and only releases the new ones out.
+///
+/// This operator does a similar inversion but for multiset semantics, with some caveats. When it
+/// receives duplicate items, instead of ignoring them, it "subtracts" them from the items received
```

Comment:
"subtracts" is confusing. Why not say "if we received `k` copies of an item in the previous tick, and we receive `l > k` copies in the current tick, we output `l - k` copies of the item."